.pipelines/stages/jobs/steps/capi-win-step.yml

```diff
@@ -46,6 +46,13 @@ steps:
   condition: eq(variables['ep'], 'cuda')
   workingDirectory: '$(Build.Repository.LocalPath)'
 
+- powershell: |
+    $exePath = "$env:TEMP\rustup-init.exe"
+    (New-Object Net.WebClient).DownloadFile('https://static.rust-lang.org/rustup/dist/x86_64-pc-windows-msvc/rustup-init.exe', $exePath)
```

Comment:
This assumes `x86_64`. Windows on ARM will not work.

Reply:
I'll revert all the azure pipeline code since it might need more discussion to decide if we want to add this to release